src/rateLimiter.js

```diff
@@ -60,6 +64,28 @@ RateLimiter.prototype.shouldSend = function(item, now) {
     this.perMinCounter = 0;
   }
 
+  if (item._isUncaught) {
+    // var item = {
+    //   message: message,
+    //   err: err,
+    //   custom: custom,
+    //   timestamp: now(),
+    //   callback: callback,
+    //   uuid: uuid4()
+    // };
+    var exceptionMessage = item.message;
+    var exception = item.err;
+    var exceptionHash = exceptionMessage + '-' + (exception.stack || String(exception));
+
+    var lastOccurrenceMS = this.dupCache[exceptionHash];
+    this.dupCache[exceptionHash] = now;
```

Comment:
The problem with this is that over time the memory footprint of `dupCache` will grow unbounded. That is not okay for server side js and it is even bad in some browser contexts as well. This is probably a scenario where an LRU cache is necessary instead of just using a js object as a cache.

Either write your own LRU cache using an object plus a doubly linked list, or I can do that, or we can pull something in from npm.